helm/files/bridge.conf

```diff
@@ -30,8 +30,8 @@ bridgeInnerConfig: {
 certificatesDirectory: "./workspace/certificates"
 sslKeystore: "./workspace/certificates/sslkeystore.jks"
 trustStoreFile: "./workspace/certificates/truststore.jks"
-keyStorePassword: "cordacadevpass"
-trustStorePassword: "{{ .Values.corda.firewall.conf.truststorePassword }}"
+keyStorePassword: "{{ .Values.corda.nodebridge.conf.nodeKeystorePassword }}"
+trustStorePassword: "{{ .Values.corda.nodebridge.conf.nodeTruststorePassword }}"
```

Comment:
No need to introduce a new nodebridge group, we already have firewall group which defines all conf changes to bridge/float conf files.